cookbooks/fb_bookworm/README.md

```diff
@@ -0,0 +1,29 @@
+fb_bookworm
+===========
+
+Bookworm is a program that gleans context from a Chef/Ruby codebase, which
+recognizes that Ruby source files in different directories have different
+semantic meaning to a larger program (ie Chef)
+
+It currently runs on top of the Chef Workstation Ruby, although there is
+nothing preventing running bookworm on vanilla Ruby using bundler, etc.
+
+Running bookworm
+----------------
```

Comment:
This should all be under a `Usage` section, these are not valid top-level headers in a cookbook README

Reply:
See above re: https://github.com/facebook/chef-cookbooks/commit/b74bf2f119b36b56510d3e8f24b0266dc5b3825d